test/db/userChallenge.test.js

```diff
@@ -0,0 +1,22 @@
+require('@babel/polyfill');
+const { UserChallenge } = require('../../server/db/models');
+const db = require('../../server/db/conn');
+
+describe('testing UserChallenge model', () => {
+    beforeAll(async () => {
+        await db.sync({force: true});
+        await UserChallenge.create({html: 'TEXT', css: 'TEXT', js: 'TEXT', grade: 90, submited: true, image: 'STRING'});
```

Comment:
will also have to be changed to an `imageId` from `image` since that also changed.  Do not have time to figure out right now.

```suggestion
        await Userchallenge.create({html: 'TEXT', css: 'TEXT', js: 'TEXT', grade: 90, submited: true, image: 'STRING'});
```